docs/tutorials/how-tos/testing-devnet-features.md

```diff
@@ -0,0 +1,203 @@
+
+{% tab label="JavaScript" %}
+
+```bash
+// Locate your ripple-binary-codec installation in node_modules and replace the definitions.json file.
+// <your project directory>/node_modules/ripple-binary-codec/dist/definitions.json
+```
+
+{% /tab %}
+
+{% tab label="Python" %}
+
+```bash
+// Use `pip show xrpl-py` to find the installation location and navigate to `<output of pip show>/xrpl/core/binarycodec/definitions/definitions.json` to replace the `definitions.json` file.
+```
+
+{% /tab %}
+
+{% /tabs %}
+
+### 4. Create and Submit Custom Transaction
+
+{% tabs %}
+
+{% tab label="JavaScript" %}
+
+```javascript
+const { Client, Wallet } = require('xrpl');
+const { encode } = require('ripple-binary-codec');
+
+async function main() {
+  const client = new Client("wss://s.devnet.rippletest.net:51233");
+  await client.connect();
+
+  const wallet = Wallet.fromSeed('sYOURSEEDHERE');
+
+  const customTx = {
+    TransactionType: 'NewTransactionType',
+    Account: wallet.address,
+    // additional fields for the new transaction
+  };
+
+  // If using Typescript, you will need to encode to allow typechecks to function
+  // or just us @ts-expect-error when calling submit
+  //   const encodedTransaction = encode(customTx);
+
+  await client.submitAndWait(customTx, { wallet });
+  // If using typescript, you should pass the encoded string of the transaction or us @ts-expect-error
+  //   await client.submitAndWait(encodedTransaction, { wallet });
+  // await client.disconnect();
+}
+
+main();
+// Or call await main(); if your nodejs versions supports top level await
+```
+
+{% /tab %}
+
+{% tab label="Python" %}
+
+```python
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import Transaction
+from xrpl.wallet import Wallet
+from xrpl.transaction import send_reliable_submission
+from xrpl.core.binarycodec import encode
+
+client = JsonRpcClient("wss://s.devnet.rippletest.net:51233")
+
+wallet = Wallet(seed="sYOURSEEDHERE", sequence=0)
+
+custom_tx = Transaction(
+    account=wallet.classic_address,
+    transaction_type="NewTransactionType",
+    # additional fields for the new transaction
+)
+
+# If using typechecking, encode the transaction into a string before passing to send_reliable_submission
+# because the new transaction type is not natively supported by xrpl-py and therefore will have a type
+# error if you pass an unsupported transaction type
+# encoded_tx = encode(custom_tx)
+
+send_reliable_submission(custom_tx, client, wallet)
+# If using typechecking, encode the transaction into a string before passing to send_reliable_submission
+# because the new transaction type is not natively supported by xrpl-py and therefore will have a type
+# error if you pass an unsupported transaction type
+# send_reliable_submission(encoded_tx, client, wallet)
+# Or disable type checking for the line
+# send_reliable_submission(custom_tx, client, wallet) # type: ignore
+```
+
+{% /tab %}
+
+{% /tabs %}
+
+### Final Considerations
```

Comment:
Suggest an alternate heading like "Key Considerations" or "Considerations for Future Development".

Reply:
What about just "considerations"?